live-editing/configs/GridConfigGenerator.ts

```diff
@@ -374,9 +375,9 @@ export class GridConfigGenerator implements IConfigGenerator {
             component: RemotePagingGridSample,
             additionalFiles: ["/src/app/grid/services/remotePagingService.ts"],
             appModuleConfig: new AppModuleConfig({
-                imports: [RemotePagingGridSample, IgxGridModule, HttpClientModule, RemotePagingService],
+                imports: [RemotePagingGridSample, IgxGridModule, RouterModule, HttpClientModule, RemotePagingService],
                 ngDeclarations: [RemotePagingGridSample],
-                ngImports: [IgxGridModule.forRoot(), HttpClientModule],
+                ngImports: [RouterModule.forRoot([]), IgxGridModule.forRoot(), HttpClientModule],
```

Comment:
Now the button is initialized, although the stackblitz sample is not loaded. Use the workaround below for the time being. Once issue #937 is fixed, we will remove this
```suggestion
                ngImports: '[RouterModule.forRoot([])', IgxGridModule.forRoot(), HttpClientModule],
```